app/__init__.py

```diff
@@ -336,6 +336,7 @@ def create_random_identifier():
     return ''.join(secrets.choice(string.ascii_uppercase + string.digits) for _ in range(16))
 
 
+# TODO maintainability what is the purpose of this?  Debugging?
```

Comment:
Noting this for the future when we take a closer look at our app config, thanks for calling it out @terrazoon!